examples/futil/simple.expect

```diff
@@ -1,18 +1,19 @@
 import "primitives/core.futil";
 import "primitives/binary_operators.futil";
-component main(@go go: 1, @clk clk: 1, @reset reset: 1) -> (@done done: 1) {
+component main<"static"=5>(@go go: 1, @clk clk: 1, @reset reset: 1) -> (@done done: 1) {
```

Comment:
Really cool effect of this pass that the component can have its static latency inferred!